diff --git a/static-build/CMakeLists.txt b/static-build/CMakeLists.txt
index 9a2f85052..bc73496e5 100644
--- a/static-build/CMakeLists.txt
+++ b/static-build/CMakeLists.txt
@@ -12,7 +12,7 @@ set(OPENSSL_VERSION 1.1.1f)
 set(ZLIB_VERSION 1.2.11)
 set(NCURSES_VERSION 6.2)
 set(READLINE_VERSION 8.0)
-set(UNWIND_VERSION 1.3-rc1)
+set(UNWIND_VERSION 1.6.2)

 # Pass -isysroot=<SDK_PATH> option on Mac OS to a preprocessor and a C
 # compiler to find header files installed with an SDK.
