diff --git a/cli/running/process.go b/cli/running/process.go
index 9b56d17..e16dd14 100644
--- a/cli/running/process.go
+++ b/cli/running/process.go
@@ -136,10 +136,6 @@ func (process *Process) SetPidAndStatus() {
 		return
 	}

-	if name != "tarantool" {
-		log.Warnf("Process %s does not seem to be tarantool", name)
-	}
-
 	if err := process.osProcess.SendSignal(syscall.Signal(0)); err != nil {
 		process.Status = procStatusStopped
 	} else {
